bin/tx-prover/README.md

```diff
@@ -1,84 +1,89 @@
 # Miden transaction prover
 
-A service for generating Miden transaction proofs on-demand. It is split in two binaries: worker and proxy.
+A service for generating Miden transaction proofs on-demand. It includes a CLI to control both workers and proxy for the Miden's remote transaction prover service, and a client, `RemoteTransactionProver` to interact with the service from a Rust codebase.
 
 The worker is a gRPC service that can receive transaction witnesses and returns the proof. It can only handle one request at a time and returns an error if is already in use.
 
 The proxy uses [Cloudflare's Pingora crate](https://crates.io/crates/pingora), which provides features to create a modular proxy. It is meant to handle multiple workers with a queue for each one. Further information about Pingora and it's features can be found in the [official GitHub repository](https://github.com/cloudflare/pingora).
 
 ## Installation
 
-Install the prover worker and proxy binaries for production using `cargo`:
+<!-- The following documentation is documented since the CLI is not on crates-io yet -->
+<!-- Install the CLI for production using `cargo`:
 
 ```sh
-cargo install miden-tx-prover-worker --locked
-cargo install miden-tx-prover-proxy --locked
+cargo install miden-tx-prover --locked
 ```
 
 This will install the latest official version of the prover. You can install a specific version using `--version <x.y.z>`:
 
 ```sh
-cargo install miden-tx-prover-worker --locked --version x.y.z
-cargo install miden-tx-prover-proxy --locked --version x.y.z
-```
-
-Both services can be installed at once from the source code using specific git revisions with `cargo install`. Note that since these aren't official releases we cannot provide much support for any issues you run into, so consider this for advanced users only.
+cargo install miden-tx-prover --locked --version x.y.z
+``` -->
 
-If you want to build from a local version, from the root of the workspace you can run:
+To build the CLI from a local version, from the root of the workspace you can run:
```

Comment:
If we want to make it less about the CLI (here and on line 30), I think we can replace "CLI" for "service" or just "crate" and then explain below how to start it.